applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -8,11 +8,14 @@
 
 #include <bluetooth/bluetooth.h>
 #include <bluetooth/audio/audio.h>
+#include <../subsys/bluetooth/audio/endpoint.h>
```

Comment:
Is this necessary? I believe that is an internal header file, and as such shouldn't be included

Reply:
If I remember correctly this is the only way of doing it now. But it should be fixed in host so that we dont need to have the internal endpoint.h included. I remember having talked to Emil about this endpoint.h file for BIS

Reply:
It is used to get access to BT_AUDIO_EP_STATE